Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -438,8 +616,123 @@ def send_asset_attributes_request(asset_id: str) -> Dict[str, Any]:
+    stored_chunks = assets_last_run.get('assets_available_chunks', [])
+    updated_stored_chunks = stored_chunks.copy()
+    export_uuid = assets_last_run.get('assets_export_uuid')
+    assets = []
+    for chunk_id in stored_chunks[:MAX_CHUNKS_PER_FETCH]:
+        assets.extend(client.download_assets_chunk(export_uuid=export_uuid, chunk_id=chunk_id))
+        # demisto.debug(f"extended assets are: {assets}")
+        updated_stored_chunks.remove(chunk_id)
+    if updated_stored_chunks:
+        assets_last_run.update({'assets_available_chunks': updated_stored_chunks,
+                                'nextTrigger': '0', "type": 1})
+    else:
+        assets_last_run.pop('nextTrigger', None)
+        assets_last_run.pop('type', None)
+        assets_last_run.pop('assets_available_chunks', None)
+        assets_last_run.pop('assets_export_uuid', None)
+    return assets, assets_last_run
+
+
+def try_get_assets_chunks(client: Client, assets_last_run):
+    """
+    If job has succeeded (status FINISHED) get all information from all chunks available.
+    Args:
+        client: Client class object.
+        assets_last_run: The last run.
+
+    Returns: All information from all chunks available.
+
+    """
+    status, chunks_available = client.get_export_assets_status(export_uuid=assets_last_run.get("assets_export_uuid"))
+    demisto.info(f'Assets report status is {status}, and number of available chunks is {chunks_available}')
+    if status == 'FINISHED':
+        assets_last_run.update({'assets_available_chunks': chunks_available})
+
+    return status
+
+
+def try_get_chunks(client: Client, export_uuid: str):
+    """
+    If job has succeeded (status FINISHED) get all information from all chunks available.
+    Args:
+        client: Client class object.
+        export_uuid: The UUID of the vulnerabilities export job.
+
+    Returns: All information from all chunks available.
+
+    """
+    vulnerabilities = []
+    status, chunks_available = client.get_vuln_export_status(export_uuid=export_uuid)
+    demisto.info(f'Report status is {status}, and number of available chunks is {chunks_available}')
+    if status == 'FINISHED':
+        for chunk_id in chunks_available:
+            vulnerabilities.extend(client.download_vulnerabilities_chunk(export_uuid=export_uuid, chunk_id=chunk_id))
+    return vulnerabilities, status
+
+
+def call_send_data_to_xsiam(assets, vulnerabilities):
+    """Sends assets and vulnerabilities to XSIAM"""
+
+    demisto.info(f"Sending {len(assets)} assets and {len(vulnerabilities)} vulnerabilities to XSIAM.")
+
+    send_data_to_xsiam(data=assets, vendor=VENDOR, product=f'{PRODUCT}_assets', data_type='assets')
+    send_data_to_xsiam(data=vulnerabilities, vendor=VENDOR, product=f'{PRODUCT}_vulnerabilities')
+
+    demisto.info("Done Sending data to XSIAM.")
+
+
+def test_module(client: Client, params):
     client.list_scan_filters()
+    if int(params.get('assetsFetchInterval')) < 60:
```

Comment:
this if can be before client.list_scan_filters()
no need for the client to call scan filters if assetsFetchInterval is less than 60 and test_module will fail on that